wordpress-mobile/WordPress-iOS
# Issues/login helpshift tweaks

This PR updates the custom tags passed to Helpshift via the login screens.  It also ensure that login related meta data is always passed to Helpshift, even when help is not accessed via an error prompt. 

To test:
If testing in the simulator you'll need to force enable Helpshift via `HelpshiftUtils checkIfHelpshiftShouldBeEnabled`

Scenario 1:
- In the login flow, navigate to one of the password screens.
- Tap the Help button in the navbar to open support
- Tap Contact Us to open a Helpshift conversation. 
- Send a test conversation and check the conversation in Helpshift to confirm it was properly tagged and the meta data was passed. 

Scenario 2: 
- Tap the more help option in the side address help dialog. 
- Start a new Helpshift conversation. 
- Send a test conversation and check the conversation in Helpshift to confirm it was properly tagged and the meta data was passed.

Scenario 3:
- Force a login error.  Dealer's choice, but one easy way is to construct an NSError and manually call the failure() block from `LoginFacade`.
- Tap the more help option in the error dialog. 
- Start a new Helpshift conversation
- Start a new Helpshift conversation. 
- Send a test conversation and check the conversation in Helpshift to confirm it was properly tagged and the meta data was passed.

Needs review: @nheagy would you kindly?
